pytest_pootle/factories.py

```diff
@@ -115,7 +115,6 @@ def fullname(self):
         # returns an incrementing index relative to the tp
         return 'Project %s' % Project.objects.count()
 
-    pootle_path = factory.LazyAttribute(lambda p: "/projects/%s" % p.code)
```

Comment:
No frigging idea :)

I think this was one we debugged together at PooCon but can't recall the reason